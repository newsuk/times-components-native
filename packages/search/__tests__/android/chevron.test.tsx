import TestRenderer from "react-test-renderer";
import Chevron from "@times-components-native/search/src/search-bar/chevron";
import React from "react";

describe("Chevron", () => {
  it("should render correctly", () => {
    const testInstance = TestRenderer.create(<Chevron onPress={() => null} />);
    expect(testInstance).toMatchSnapshot();
  });
});
