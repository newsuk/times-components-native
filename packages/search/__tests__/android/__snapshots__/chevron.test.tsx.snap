// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chevron should render correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <RNSVGSvgView
    align="xMidYMid"
    bbHeight="24"
    bbWidth="24"
    fill="none"
    focusable={false}
    height="24"
    meetOrSlice={0}
    minX={0}
    minY={0}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "borderWidth": 0,
        },
        Object {
          "flex": 0,
          "height": 24,
          "width": 24,
        },
      ]
    }
    vbHeight={24}
    vbWidth={24}
    width="24"
  >
    <RNSVGGroup
      fill={null}
      propList={
        Array [
          "fill",
        ]
      }
    >
      <RNSVGMask
        height="24"
        maskContentUnits={1}
        maskTransform={
          Array [
            1,
            0,
            0,
            1,
            0,
            0,
          ]
        }
        maskUnits={1}
        name="mask0"
        width="24"
        x="0"
        y="0"
      >
        <RNSVGPath
          d="M15.41 7.41L14 6L8 12L14 18L15.41 16.59L10.83 12L15.41 7.41Z"
          fill={-16777216}
          propList={
            Array [
              "fill",
            ]
          }
        />
      </RNSVGMask>
      <RNSVGGroup
        mask="mask0"
      >
        <RNSVGRect
          fill={-16777216}
          height="24"
          propList={
            Array [
              "fill",
            ]
          }
          width="24"
          x={0}
          y={0}
        />
      </RNSVGGroup>
    </RNSVGGroup>
  </RNSVGSvgView>
</View>
`;
